page/docs/guide/postman.md

```diff
@@ -50,6 +50,45 @@ constructor(MockStarknetMessaging mockStarknetMessaging_) public {
 }
 ```
 
+### Postman - l1 to l2 mock endpoint
+
+```
+POST /postman/send_message_to_l2
+```
+Sending mock transactions from L1 to L2 without the need for running L1.
+Deployed L2 contract address `l2_contract_address` and `entry_point_selector` must be valid otherwise new block will not be created.
+
+Example POST json:
+```
+{
+    "l2_contract_address":"0x00285ddb7e5c777b310d806b9b2a0f7c7ba0a41f12b420219209d97a3b7f25b2",
+    "entry_point_selector":"0xC73F681176FC7B3F9693986FD7B14581E8D540519E27400E88B8713932BE01",
+    "l1_contract_address":"0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
+    "payload":[
+      "0x1",
+      "0x2"
+    ],
+    "nonce":"0x0"
+}
+```
+
+### Postman - l2 to l1 mock endpoint
+
+```
+POST /postman/send_message_to_l1
+```
+Sending mock transactions from L2 to L1.
```

Comment:
I know it says `mock`, but I find this misleading. I would really expect something to be sent to L1. With sending to L2 it's different because there we are actually sending something to L2.

Perhaps `/consume_message_from_l2`?

Reply:
ture, I'll fix that

I just change it from `send_message_to_l2` to `send_message_to_l1` without thinking too much but actually, there is a difference there

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/365/commits/ab6b0321b6f97970bfa31bce95af24e24b1edeff